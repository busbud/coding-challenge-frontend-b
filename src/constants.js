export const BLUE = "#127ccb";