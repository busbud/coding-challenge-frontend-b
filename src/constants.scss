$action: #165fcc;
$primary: #114f91;
$secondary: #6f35b4;
$minor: #6b7072;
$major: #1f242b;
$bg: #f6f6f6;

$small: 10px;
$smaller: 5px;
$normal: 15px;
$big: 20px;
$bigger: 30px;

$line-height-base: 1.428571429;

$screen-xs-min: 480px;

// Small screen / tablet
$screen-sm-min: 768px;

// Medium screen / desktop
$screen-md-min: 992px;

// Large screen / wide desktop
$screen-lg-min: 1200px;

// Non-overlap
$screen-xs-max: ($screen-sm-min - 1);
$screen-sm-max: ($screen-md-min - 1);
$screen-md-max: ($screen-lg-min - 1);
