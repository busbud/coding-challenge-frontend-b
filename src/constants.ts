export const DATE_FORMAT_LIST = 'YYYY-MM-DD';
export const TIME_FORMAT_LIST = 'hh:mm a';