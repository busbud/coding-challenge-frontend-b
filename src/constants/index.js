const EN = {
  header: "Let's go to Osheaga!",
  on: 'departure date',
  day: 'August 2 2019',
  outboundDate: '2019-08-02',
  from: 'departing from',
  origin: 'dr5reg',
  nyc: 'New York',
  to: 'arriving to',
  mtl: 'Montreal',
  destination: 'f25dvk',
  departureTimeLabel: 'Departure Time',
  departureAddressLabel: 'Depature Address',
  arrivalTimeLabel: 'Arrival Time',
  arrivalAddressLabel: 'Arrival Address',
  travelTimeLabel: 'Travel Time',
  operatorLabel: 'Operator',
  priceLabel: 'Price',
}

const FR = {
  header: "Allons au festival d'Osheaga!",
  on: 'date de départ',
  day: '2 août 2019',
  outboundDate: '2019-08-02',
  from: 'au départ de',
  origin: 'dr5reg',
  nyc: 'New York',
  to: 'arriver à',
  mtl: 'Montréal',
  destination: 'f25dvk',
  departureTimeLabel: 'Heure de départ',
  departureAddressLabel: 'Adresse de départ',
  arrivalTimeLabel: 'Arrival Time',
  arrivalAddressLabel: 'Depature Address',
  travelTimeLabel: 'Temps de voyage',
  operatorLabel: 'Affaires',
  priceLabel: 'Prix',
}

export default { translations: { EN, FR } }
