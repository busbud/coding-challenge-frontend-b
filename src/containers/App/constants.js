export const LOCALES = ['en', 'fr'];

export const DEFAULT_LOCALE = 'en';

export const UPDATE_QUERY = '@@/App/UPDATE_QUERY';

export const FETCH_DEPARTURES = '@@/App/FETCH_DEPARTURES';

export const FETCH_DEPARTURES_SUCCESS = '@@/App/FETCH_DEPARTURES_SUCCESS';

export const FETCH_DEPARTURES_ERROR = '@@/App/FETCH_DEPARTURES_ERROR';

export const FETCH_DEPARTURES_COMPLETE = '@@/App/FETCH_DEPARTURES_COMPLETE';
