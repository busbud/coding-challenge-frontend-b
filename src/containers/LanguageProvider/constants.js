export const CHANGE_LOCALE = '@@/LanguageProvider/CHANGE_LOCALE';
