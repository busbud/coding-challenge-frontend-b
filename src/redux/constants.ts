export enum Reducers {
    DEPARTURE_REDUCER = 'DEPARTURE_REDUCER',
}