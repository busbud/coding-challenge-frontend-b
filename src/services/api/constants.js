export const BASE_URL = 'https://napi.busbud.com';

export const ACCEPT = 'application/vnd.busbud+json; version=2; profile=https://schema.busbud.com/v2/';
