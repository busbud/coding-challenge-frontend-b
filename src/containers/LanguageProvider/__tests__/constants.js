import * as Constants from '../constants';

describe('containers | LanguageProvider | constants', () => {
  it('should be defined', () => {
    // then
    expect(Constants).toBeDefined();
  });

  it('should match snapshot', () => {
    // then
    expect(Constants).toMatchSnapshot();
  });
});
