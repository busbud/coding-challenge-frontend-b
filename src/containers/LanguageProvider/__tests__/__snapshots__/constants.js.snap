// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers | LanguageProvider | constants should match snapshot 1`] = `
Object {
  "CHANGE_LOCALE": "@@/LanguageProvider/CHANGE_LOCALE",
}
`;
